features/android/support/helper.rb

```diff
@@ -0,0 +1,19 @@
+=begin
+
+These methods are implemented for Calabash-ios.
+We are implementing their Android equivalents if possible so some of them may be empty.
+
+=end
+
```

Comment:
Wrap these method definitions in a module to avoid defining the methods in the ruby kernel and/or clobbering the iOS versions.

Rename this file as `patches.rb` and add the following:

```
require "calabash-android"

module Calabash
  module Android
    module WaitHelpers
       def wait_for_none_animating(*args)
         sleep 1
       end
    end

    module Operations
      def scroll_to_row_with_mark(text)
        query_text = "* {text CONTAINS[c] '#{text}'}"
        while(!check_if_element_exists(query_text))
        scroll("android.support.v4.widget.NestedScrollView", :down)
        sleep 0.2
     end
  end
```